releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,6 +4,86 @@
+* Added netstandard 2.1 as a target framework in the Microsoft.FeatureManagement package. https://github.com/microsoft/FeatureManagement-Dotnet/pull/267
+* Added net7.0 as a target framework in the Microsoft.FeatureManagement.AspNetCore package. https://github.com/microsoft/FeatureManagement-Dotnet/pull/267
+
+## Bug Fixes
+* Prevents the usage of colon in Feature names.
+* Adjusts log level for noisy warning when feature definitions are not found.
+* Fixed an edge case in targeting if a user is allocated to exactly the 100th percentile (~1 in 2 billion chance)
+
+### Migration
+
+#### Adding built-in filters
+
+It is no longer necessary to register the following filters manually:
+
+  * `TimeWindowFilter`
+  * `ContextualTargetingFilter`
+  * `PercentageFilter`
+
+The following code:
+
+```
+services.AddFeatureManagement()
+    .AddFeatureFilter<TimeWindowFilter>();
+```
+
+should be simplified to:
+
+```
+services.AddFeatureManagement();
+```
+
+#### Adding Targeting Filter
+
+Since the `TargetingFilter` (the non-contextual version) requires an implementation of ITargetingContextAccessor to function, it is not added by default. However, a discovery/helper method was added to streamline it's addition.
+
+The following code:
+
+```
+services.AddSingleton<ITargetingContextAccessor, MyTargetingContextAccessor>();
+
+services.AddFeatureManagement()
+    .AddFeatureFilter<TargetingFilter>();
+```
+
+should be simplified to:
+
+```
+services.AddFeatureManagement()
+    .WithTargeting<MyTargetingContextAccessor>();
+```
+
+### Additional
+
+#### Contextual/non-contextual filter selection process
+The following passage describes the process of selecting a filter when a contextual and non-contextual filter of the same name are registered in an application.
+
+Let's say we have a non-contextual filter called FilterA and two contextual filters FilterB and FilterC which accept TypeB and TypeC contexts respectively. All of three filters share the same alias "SharedFilterName".
+We also have a feature flag "MyFeature" which uses the feature filter "SharedFilterName" in its configuration.
+
+If all of three filters are registered:
+When we call IsEnabledAsync("MyFeature"), the FilterA will be used to evaluate the feature flag.
+When we call IsEnabledAsync("MyFeature", context), if context's type is TypeB, FilterB will be used and if context's type is TypeC, FilterC will be used.
+When we call IsEnabledAsync("MyFeature", context), if context's type is TypeF, FilterA will be used.
```

Comment:
All mixed in one line in md. You may want to add bullet points in front of each line.